scripts/deploy-helm.sh

```diff
@@ -359,6 +359,11 @@ if [[ $CREATE_AUTOPILOT_CLUSTER == "Y" ]]; then
     err "Default region not set. Set default region by running 'gcloud config set compute/region <REGION>'."
     exit 1
   fi
+  CONTAINERS_API=$(gcloud services list --enabled --filter=container.googleapis.com --project="$GCP_PROJECT")
+  if [[ -z "$CONTAINERS_API" ]]; then
+    debug "Enabling Containers API"
```

Comment:
Shouldn't this be Kubernetes Engine API?

Reply:
Yes, that's the name Google uses for the service:
https://cloud.google.com/kubernetes-engine/docs/reference/rest/#:~:text=Kubernetes%20Engine%20API%20Builds%20and%20manages%20container-based,applications%2C%20powered%20by%20the%20open%20source%20Kubernetes%20technology.

Reply:
I mean that the debug message says "Enabling Containers API"

Reply:
Ok, changed to be more accurate